app/vendors/shells/sphinx_conf.php

```diff
@@ -240,7 +240,7 @@ function($v) { return $v != 'A..Z->a..z' && $v != 'a..z'; }
          * to search if they are ignored. Since all the Russian diacritics
          * are not single characters but combining characters (e.g. и + ´ = и́)
          * we simply ignore the *´* combining char (U+301) so that characters
-         * are considered not having a diacritic. */
+         * are considered as not having a diacritic. We also ignore soft hyphen. */
```

Comment:
Note that we ignore soft hyphen here only because we ignore them in every language: the parent config block `common_index` says `ignore_chars = U+AD`. Because of the way Sphinx config is constructed, whenever you need to append other values to a config variable in a child block like here, you need to repeat the whole value of the parent. In other words
```php
index parent {
  foo = bar
}
index child : parent {
  foo = baz # This would make the value to be assigned 'baz' only.
  foo = bar, baz # This is what we actually want.
}
```

Now I see that the `ignore_chars` option of `common_index` has some other values, namely `U+5B0..U+5C5, U+5C7`. So you want to put them here too, or do something to avoid code duplication.